assessment.js

```diff
@@ -1,8 +1,33 @@
-//We'll learn about require later in the course
+5 Primitives
+    Number  -0, NaN, Infinity
+    String '', "non-empty String",
+    Boolean - true, false
+    Null - Null
+    Undefined - Undefined
+
+    Question 2
+    Value of c is 7
+
+
+    Operator is '+' where c is combined to d using the variable assigned.
+
+    Question 3
+    x = 3
+    y = 10
+    yes the variables change
+    Their values change
+
+    Question 4
+    The value of weather = "sunny"; is sunny
+    The value of weather === "sunny" is true
+
+    Question 5
+    //We'll learn about require later in the course
 var ask = require('./ask.js');
 
 var answer = 'not empty';
 
-while (answer !== '' && answer !== 'SeCrEt') {
-  answer = ask("Guess my secret? ");
+while (answer >= '1' && answer <= '10') {
+  answer = ask("Correct ");
```

Comment:
The prompt is a little unhelpful, but this code will otherwise work as specified. Nice job!